group_boards.go

```diff
@@ -136,6 +136,66 @@ func (s *GroupIssueBoardsService) GetGroupIssueBoard(gid interface{}, board int,
 	return gib, resp, err
 }
 
+type UpdateGroupIssueBoardOptions struct {
+	ID          *string `url:"id" json:"id"`
+	BoardID     *int    `url:"board_id" json:"board_id"`
```

Comment:
`ID` and `BoardID` are already in the function signature (`gid` and `board`), so they should be taken out of the options struct.